.github/workflows/build-test-release.yml

```diff
@@ -6,14 +6,6 @@ name: Build and Test C++ and Python
 
 # Controls when the workflow will run
 on:
-  # run pipeline on push event of main branch
-  push:
-    branches:
-      - main
-  # run pipeline on pull request
-  pull_request:
-  # run pipeline on merge queue
-  merge_group:
```

Comment:
cfr. discussion: please prove that releasing still works correctly on your own fork

Reply:
proved